people.qmd

```diff
@@ -0,0 +1,724 @@
+</td>
+</tr>
+
+
+<tr>
+<td align="center">
+<a href="https://github.com/oespinozah">
+<img src="https://avatars.githubusercontent.com/u/60980297?u=49afaa50e6667b1995bd4c6610c54dc0188937c6&v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=oespinozah">oespinozah</a>
+</td>
+<td align="center">
+<a href="https://github.com/BlackEdder">
+<img src="https://avatars.githubusercontent.com/u/845741?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=BlackEdder">BlackEdder</a>
+</td>
+<td align="center">
+<a href="https://github.com/GeraldineGomez">
+<img src="https://avatars.githubusercontent.com/u/46997659?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=GeraldineGomez">GeraldineGomez</a>
+</td>
+<td align="center">
+<a href="https://github.com/erees">
+<img src="https://avatars.githubusercontent.com/u/22442251?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=erees">erees</a>
+</td>
+<td align="center">
+<a href="https://github.com/agreene71">
+<img src="https://avatars.githubusercontent.com/u/11233984?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=agreene71">agreene71</a>
+</td>
+</tr>
+
+
+<tr>
+<td align="center">
+<a href="https://github.com/AKuederle">
+<img src="https://avatars.githubusercontent.com/u/10776763?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=AKuederle">AKuederle</a>
+</td>
+<td align="center">
+<a href="https://github.com/annajiat">
+<img src="https://avatars.githubusercontent.com/u/3046889?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=annajiat">annajiat</a>
+</td>
+<td align="center">
+<a href="https://github.com/BernhardKonrad">
+<img src="https://avatars.githubusercontent.com/u/2601674?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=BernhardKonrad">BernhardKonrad</a>
+</td>
+<td align="center">
+<a href="https://github.com/daisieh">
+<img src="https://avatars.githubusercontent.com/u/1827728?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=daisieh">daisieh</a>
+</td>
+<td align="center">
+<a href="https://github.com/ErinBecker">
+<img src="https://avatars.githubusercontent.com/u/19176319?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=ErinBecker">ErinBecker</a>
+</td>
+</tr>
+
+
+<tr>
+<td align="center">
+<a href="https://github.com/fmichonneau">
+<img src="https://avatars.githubusercontent.com/u/5502922?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/epiverse-trace/varnish/commits?author=fmichonneau">fmichonneau</a>
```

Comment:
A bunch of users here didn't really contribute to Epiverse. They come from the https://github.com/epiverse-trace/git-rstudio-basics repo, which was forked from a Carpentries lesson. Can we filter by date (= start at the starting date of the project) or by repo?

Reply:
We can exclude certain repositories altogether 👍 If we want to filter within the contributions of a specific repo, I will have to see whether I can extend the `allcontributors` package to allow for this (they seem to be very receptive as I already started adding some functionality - see https://github.com/ropenscilabs/allcontributors/issues/34)